deploy/crds/containerized-data-importer00.crd.yaml

```diff
@@ -82,8 +84,10 @@ spec:
               type: string
           type: object
       type: object
-  version: v1alpha1
   versions:
-  - name: v1alpha1
+  - name: v1beta1
     served: true
     storage: true
+  - name: v1alpha1
```

Comment:
Is it working? in v1.20.0, v1alpha1 does not exists

Reply:
this manifest is generated by build-manifests.sh --> csv-generator

Reply:
this is coming from:
https://github.com/kubevirt/containerized-data-importer/blob/master/pkg/operator/resources/operator/operator.go#L301